src/client/components/StudyGroupLogo/StudyGroupLogo.stories.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { withKnobs } from '@storybook/addon-knobs';
+
+import { StudyGroupLogo } from './StudyGroupLogo';
+import HYFLogo from './HYF.logo.svg';
+import ZendeskLogo from './Zendesk.logo.svg';
+import ElasticLogo from './Elastic.logo.svg';
+
+export default {
+  title: 'StudyGroupLogo',
+  decorators: [withKnobs],
```

Comment:
Remove knobs if you don't need them.

Reply:
removed

Reply:
Great job and nice that you removed knobs as you don't use them.